sci-astronomy/stellarium/stellarium-25.3.recipe

```diff
@@ -62,9 +62,12 @@ BUILD_REQUIRES="
 	devel:libz$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
+	llvm20
```

Comment:
This should be `llvm20$secondaryArchSuffix`

Reply:
Probably same suffix should be added for cmd:clang++ too?